pyproject.toml

```diff
@@ -1,3 +1,6 @@
+[tool.black]
+exclude = "/(\\.direnv|\\.eggs|\\.git|\\.hg|\\.mypy_cache|\\.nox|\\.tox|\\.venv|venv|\\.svn|\\.ipynb_checkpoints|_build|buck-out|dist|__pypackages__)/"
```

Comment:
Remove `build` from [default](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#command-line-options)

Reply:
Maybe add a comment saying that.

Reply:
OK.
Added comment!